recipes/enable-postfix-smarthost.rb

```diff
@@ -49,6 +53,19 @@
   retry_delay 5
 end
 
+# creates a file containing both key and cert
+execute 'create dummy key/cert' do
+  command %Q|/etc/pki/tls/certs/make-dummy-cert #{postfix_cert_path}|
+  retries 5
+  retry_delay 5
+end
+
+file postfix_cert_path do
+  mode '400'
+  owner 'root'
+  group 'root'
+end
+
```

Comment:
On debian/ubuntu every installation includes a set of "snakeoil" cert/key files for this purpose. On amazonlinux we need to create our own. This allows the postfix smtpd component to enable TLS handling on incoming mail. I'm not 100% why we need that since we're only handling internally-generated messages, but we set it up that way originally so I guess we'll keep it working.